config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -0,0 +1,223 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
look at, consider and delete file if not needed

Reply:
@colinbruce we could either keep this file (generated by `bin/rails app:update`) which only has one uncommented option in there. or could remove, placing the single config into the main config file equivalent. I quite like having the details setout in there but it is a bit manky. What do you think?

Reply:
It _does_ add a lot of bloat... maybe add a reminder to move it over into application.rb after 2 months if no changes are needed?